gusto/linear_solvers.py

```diff
@@ -156,14 +159,26 @@ def V(u):
         dxp = dx(degree=(self.quadrature_degree))
         dS_vp = dS_v(degree=(self.quadrature_degree))
 
+        # introduce density potential temperatures
+        theta_rho = theta
```

Comment:
I think it is especially confusing because elsewhere we use ```exner_rho``` to mean d(exner)/d(rho) i.e. the derivative of the Exner pressure w.r.t. rho so this kind of looks like d(theta/d(rho).

Can you not just keep it ```theta``` and then expand your comment "add moisture fields" to explain that when there is moisture, theta becomes blah...?